data_dir/scylla.repo.upgrade_1.6

```diff
@@ -0,0 +1,23 @@
+[scylla]
+name=Scylla for Centos $releasever - $basearch
+baseurl=https://s3.amazonaws.com/downloads.scylladb.com/rpm/centos/scylladb-1.6/$releasever/$basearch/
+enabled=1
+gpgcheck=0
+
+[scylla-generic]
+name=Scylla for centos $releasever
+baseurl=https://s3.amazonaws.com/downloads.scylladb.com/rpm/centos/scylladb-1.6/$releasever/noarch/
```

Comment:
We have a backend service to generate and return valid repo content, maybe we can directly use that.  @roydahan @abvgedeika